Backend/src/test/java/ca/mcgill/cooperator/service/CooperatorServiceNotificationTests.java

```diff
@@ -54,6 +54,11 @@ public void testCreateNotification() {
         }
 
         assertEquals(1, notificationService.getAllNotifications().size());
+        student = studentService.getStudentById(student.getId());
+        assertEquals("Hello", ((Notification)student.getNotifications().toArray()[0]).getTitle());
```

Comment:
Why the cast to Notification?

Reply:
because toArray() from a set returns an array of object so you need to cast